Sources/TarkaUI/EnvironmentKey/SafeArea/SafeAreaEnvironment.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SafeAreaEnvironment.swift
+//  EAM360
+//
+//  Created by Gopinath on 20/11/25.
+//  Copyright © 2025 Sedin Technologies. All rights reserved.
+//
+
+import SwiftUI
+
+private struct SafeAreaKey: EnvironmentKey {
+    static let defaultValue: EdgeInsets = .init()
+}
+
+public extension EnvironmentValues {
+  var tuiSafeAreaInsets: EdgeInsets {
```

Comment:
```suggestion
  @Entry var tuiSafeAreaInsets: EdgeInsets = .init()
```
`SafeAreaKey` struct is not required while using `@Entry macro`